stdlib/runtime/wasi.gr

```diff
@@ -77,10 +77,10 @@ provide foreign wasm fd_prestat_dir_name: (
 /**
  * Invokes the `fd_write` system call.
  *
- * @param file_descriptor: The file descriptor to write to
- * @param iovs: The pointer to the array of iovs to write
- * @param iovs_len: The length of the array of iovs
- * @param nwritten: Where to store the number of bytes written
+ * @param 0: The file descriptor to write to
```

Comment:
Same question about primitives, does it also apply to foreigns?

Reply:
Actually I don't think there's anything prohibiting using named arguments on foreigns, I think we just haven't gotten around to assigning argument names to all of the foreigns in the stdlib. I can do this in this PR if you'd like or alternatively in a followup PR

Reply:
#1982